src/pages/blog/5-tips-for-actually-shipping-a-side-project.pug

```diff
@@ -7,81 +7,86 @@ block prepend vars
   -var metaDescription = "How to ensure your side project actually ships, and gets worked on consistently."
 
   -var heroHeading = "5 tips for actually shipping a side project"
-  -var heroSubheading = "(by Mike Carter)"
 
 block pageComponents
 
-  +banner({
-  })
+  article
 
-  .banner-spacer
+    +hero({
+      heroHeading: heroHeading,
+      heroAuthor: "Mike Carter",
+      heroDate: "20 March 2017",
+      heroMinutesToRead: 4
```

Comment:
Not a blocker, but isn't the "hero" prefix on the attributes being passed in here redundant? It's already implicit they're for the hero by the fact they're passed to the hero.

Reply:
I've thought about this a few times, and it's for no better reason than it just follows the convention of the other components. If you're happy for us to, we'll do a sweep of all components and make their attribute names a little nicer.